### PR TITLE
Add summarize API

### PR DESCRIPTION
**Pull request by mkozakov**
This PR adds support for the new Summarize API

### CODE REVIEW

File: summarize.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package cohere
+
+type SummarizeOptions struct {
+	// Texts to identify languages for
```
Comment (abdullahkady) [inline summarize.go:4]: I think this comment is missing?

File: summarize.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package cohere
+
+type SummarizeOptions struct {
```
Comment (abdullahkady) [inline summarize.go:3]: where is `extractiveness`? also the comments seem to be less explanatory compared to node's for example. can you sync them please?

**Review by abdullahkady — CHANGES_REQUESTED** [review]

File: summarize.go (hunk 2 above)
Comment (mkozakov) [reply summarize.go:3]: Good catch. updated

File: summarize.go (hunk 1 above)
Comment (mkozakov) [reply summarize.go]: 👍

**Pull request merged**